### PR TITLE
Release notes 6.6.0

### PR DESCRIPTION
**Pull request by mwz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Changelog
 ===
+## [6.6.0](https://github.com/mwz/sonar-scala/releases/tag/v6.6.0) - ??.09.2018
+- Refactored Scalastyle module and introduced new Scalastyle rules repository along with a new quality profile, which now in addition to the latest Scalastyle rules also consists of rule templates which are activated by default. The existing Scalastyle quality profile and rule registry have been deprecated and will be removed in the next major version of the plugin (7.x). The new module also exposes the following optional property `sonar.scala.scalastyle.disable`, which allows you to disable the Scalastyle sensor. (#35 - @mwz)
+- New `Scalastyle+Scapegoat` quality profile, which includes all of the default rules from Scalastyle and Scapegoat quality profiles. (#112 - @BalmungSan)
+- Bumped Scapegoat up to `1.3.7`, which introduces a new inspection ([UnsafeStringContains](https://github.com/sksamuel/scapegoat/blob/v1.3.7/src/main/scala/com/sksamuel/scapegoat/inspections/string/UnsafeStringContains.scala)) and other various fixes and improvements - see the [diff](https://github.com/sksamuel/scapegoat/compare/v1.3.5...v1.3.7) for more details. (#100 - @mwz)
```
Comment (BalmungSan) [inline CHANGELOG.md]: I think we should do a general dumping of dependencies before the release, especially **classgraph** _(see [this](https://github.com/mwz/sonar-scala/pull/110#discussion_r217233297) comment)_, and **Scapegoat** to [`1.3.8`](https://github.com/sksamuel/scapegoat/releases/tag/v1.3.8).

File: README.md [hunk 2]
```diff
@@ -50,14 +50,11 @@ In addition to the above, the plugin reports the following custom metrics, which
 
 
 # Quality Rules and Profiles
-This plugin integrates 69 quality checks from [Scalastyle](http://www.scalastyle.org/rules-1.0.0.html) and 125 from [Scapegoat](https://github.com/sksamuel/scapegoat/tree/v1.3.5). 40 of the Scalastyle rules are defined without parameters and work out of the box and the remaining 29 are rule templates that allow you to set up custom rules by specifying the parameters yourself. Scapegoat rules don't contain any templates.
+This plugin integrates 69 quality checks from [Scalastyle](http://www.scalastyle.org/rules-1.0.0.html) and 126 from [Scapegoat](https://github.com/sksamuel/scapegoat/tree/v1.3.5). 40 of the Scalastyle rules are defined without parameters and the remaining 29 are rule templates, which allow you to set up custom rules by specifying the parameters yourself. Most of those are are already activated for you with default values recommended by Scalastyle. Scapegoat rules don't contain any templates.
```
Comment (BalmungSan) [inline README.md]: Update the **Scapegoat** link.
And maybe, add link to [**Scalastyle**](http://www.scalastyle.org/rules-1.0.0.html).

File: CHANGELOG.md (hunk 1 above)
Comment (mwz) [reply CHANGELOG.md]: Scapegoat 1.3.8 doesn't add anything that would be of interest to us so I'll skip it for now and I'll bump up the rest in a separate PR.

File: README.md (hunk 2 above)
Comment (mwz) [reply README.md]: Where would you like to add the link to Scalastyle? It's already [here](https://github.com/mwz/sonar-scala/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8R53) and [here](https://github.com/mwz/sonar-scala/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8R60).

File: CHANGELOG.md (hunk 1 above)
Comment (BalmungSan) [reply CHANGELOG.md]: Ok

File: README.md (hunk 2 above)
Comment (BalmungSan) [reply README.md]: Nah forget it, I probably was sleepy and didn't see it.

**Pull request merged**